Src/BootCamp.Chapter/Solution.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+
+public class Solution
+{
+    public int MyAtoi(string str)
```

Comment:
😞 Why is there no task of what you're doing? How am I supposed to know. What is Atoi, what are you even trying to do...?

Reply:
Please learn how to name things properly. MyAtoi is a meaningless name to me.
It's the 100th time I repeat myself to name methods as VERBS and not NOUNS!

Reply:
No commnents, no nothing and I am left clueless.